rest-service/manager_rest/test/base_test.py

```diff
@@ -211,23 +213,19 @@ def _handle_flask_app_and_db(self, server):
         self._handle_default_db_config(server)
         self._setup_anonymous_user(server.app, server.user_datastore)
 
-    def _init_default_tenant(self, db, app):
```

Comment:
No longer needed - this is being taken care of in `create_default_user_tenant_and_roles`